opus/application/static_media/js/browse.js

```diff
@@ -102,17 +102,22 @@ var o_browse = {
         });
 
         // browse nav menu - download csv
-        $("#browse").on("click", ".download_csv", function() {
-            let col_str = opus.prefs.cols.join(',');
-            let hash = [];
+        $("#browse").on("click", ".op-download-csv", function() {
+            let colStr = opus.prefs.cols.join(',');
+            let selectionsHash = [];
             for (let param in opus.selections) {
                 if (opus.selections[param].length) {
-                    hash[hash.length] = param + '=' + opus.selections[param].join(',').replace(/ /g,'+');
+                    selectionsHash.push(param + '=' + opus.selections[param].join(',').replace(/ /g,'+'));
```

Comment:
might as well change this one to `${}` format as well

Reply:
I didn't want to put that whole complicated replace thing inside a {}. I can make it another variable.